ledger/sandbox-on-x/reference.conf

```diff
@@ -562,19 +562,6 @@ ledger {
         submission-batch-size = 50
       }
 
-      lf-value-translation-cache {
```

Comment:
@skisel I assume we need to delete this. Is there something else to do if we are removing configurations?

Reply:
I don't see any other usage, so it's okay

Reply:
btw, you've tagged a wrong skisel ;)